pipeline_steps/delivery.py

```diff
@@ -0,0 +1,11 @@
+from datetime import datetime
+import pytz, logging
+from base import PipelineStep
+class PipelineDelivery(PipelineStep):
+    @staticmethod
+    def process_message(message):
+        logging.warning(message.current_step + " has received " + message.name + " " + str(message.id))
+        if pytz.utc.localize(datetime.utcnow()) > message.expiration_time:
+            logging.warning(message.name + " " + str(message.id) + " has expired. It will be dropped.")
```

Comment:
super(PipelineDelivery) is incorrect usage apparently. See http://stackoverflow.com/questions/26788214/super-and-staticmethod-interaction. Used super(PipelineDelivery, PipelineDelivery) instead. Might remove the static method later to prevent causing problems downstream.